### ISSUE TITLE
Lib doesn't build due to a change in serde

### ISSUE THREAD

**Issue by fafk** [rust-db/refinery#150]
Serde changed the visibility of `mod export` from public to private and it breaks refinery.

https://github.com/serde-rs/serde/commit/dd1f4b483ee204d58465064f6e5bf5a457543b54 line 275

```
error[E0603]: module `export` is private
   --> /Users/fafk/.cargo/registry/src/github.com-1ecc6299db9ec823/refinery-core-0.3.2/src/runner.rs:11:12
    |
11  | use serde::export::Formatter;
    |            ^^^^^^ private module
    |
note: the module `export` is defined here
   --> /Users/fafk/.cargo/registry/src/github.com-1ecc6299db9ec823/serde-1.0.119/src/lib.rs:275:5
    |
275 | use self::__private as export;
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0603`.
error: could not compile `refinery-core`
```

**Comment by jxs** [comment]
Hi, this has been addressed in master: https://github.com/rust-db/refinery/pull/149, but i will release a fix ASAP.
thanks

**Comment by jxs** [comment]
released `refinery_core:0.5.1`   
do `cargo update` to update `Cargo.lock` to use it
